src/document.rs

```diff
@@ -195,13 +195,19 @@ impl Document {
                                 text
                             } else {
                                 // no text event, just Event::End
-                                Cow::Borrowed("").into()
+                                Cow::Borrowed("#").into()
```

Comment:
The intent here was to have *something* that can be clicked on if there is no label.  I picked a "#" for no good reason.  :)

Come to think of it... this is where the FontAwesome "▶️" should be!  Thoughts on that?

Reply:
I'd probably make the FontAwesome change in another commit.  The current fix resolves the case where there is no text.  

TBH I should have split this into two PRs - one for FontAwesome and one for playing audio in-line.  Next time.  :)

Reply:
I assumed CSS with padding and maybe a circle around the triangle (which can be done with pure css or FontAwesome has a circle-play glyph

Reply:
Aaah, I made this change before I did the in-line media player.  I'm not totally sure how to test this now because all modern browsers support in-line audio players.  Maybe test when we get generation of `.epub` working...